src/container/srv_target.c

```diff
@@ -1375,6 +1376,13 @@ cont_child_aggregate(struct ds_cont_child *cont)
 	if (epoch_max >= cont->sc_aggregation_max)
 		epoch_max = cont->sc_aggregation_max - 1;
 
+	/* Aggregate until rebuild stable epoch. TODO: actually
+	 * we can skip this epoch XXX
+	 */
+	if (cont->sc_pool->spc_rebuild_epoch > 0 &&
+	    epoch_max >= cont->sc_pool->spc_rebuild_epoch)
+		epoch_max = cont->sc_pool->spc_rebuild_epoch;
```

Comment:
Given that the last rebuild stable view is EPOCH_MAX, why do we need this?

Reply:
The last stable view is not EPOCH_MAX, it should be this rebuild_epoch marked during rebuild start.
the current patch will make aggregation blocking by the stable view until the rebuild finish.

Reply:
In this patch, we use EPOCH__MAX as the last stable view, right?

Reply:
you are right. I should not use EPOCH_MAX in the patch. hmm

Reply:
ah, yes, but I can not block the aggregation by MAX?

Reply:
I mean since rebuild fetch data from epoch MAX, capping aggregation to this 'spc_rebuild_epoch' looks not necessary to me.

Reply:
oh, I actually will change the epoch MAX to spc_rebuild_epoch in the following patch,  but need to think a bit.

Reply:
we need to fetch the data from spc_rebuild_epoch, instead of MAX. I just do not know how to create an unique global epoch.